src/Model/Post/Media.php

```diff
@@ -616,6 +621,63 @@ public static function getType(string $mimeType): int
 		return $type;
 	}
 
+	/**
+	 * Fetch video dimensions using getID3
+	 *
+	 * @param array $media
+	 * @return array media with added dimensions
+	 */
+	private static function getVideoDimensions(array $media): array
+	{
+		if (!empty($media['width']) && !empty($media['height'])) {
+			return $media;
+		}
+
+		DI::logger()->debug('Fetch video dimensions', ['uri-id' => $media['uri-id'], 'url' => $media['url']]);
+		$timestamp  = microtime(true);
+		$timeout    = DI::config()->get('system', 'xrd_timeout');
+		$curlResult = DI::httpClient()->get($media['url'], HttpClientAccept::AS_DEFAULT, [HttpClientOptions::TIMEOUT => $timeout, HttpClientOptions::HEADERS => ['Range' => 'bytes=0-1000000']]);
+		if (!$curlResult->isSuccess()) {
+			DI::logger()->notice('Could not fetch video', ['uri-id' => $media['uri-id'], 'url' => $media['url'], 'code' => $curlResult->getReturnCode()]);
+			return $media;
+		}
+
+		if (empty($media['modified']) && !empty($curlResult->getHeader('Last-Modified')[0])) {
+			$media['modified'] = DateTimeFormat::utc($curlResult->getHeader('Last-Modified')[0]);
+			if (empty($media['published'])) {
```

Comment:
```suggestion
			if (!isset($media['published'])) {
```
I guess ANY value is ok, or is `0` wrong as well (or even possible?)

Reply:
I changed it.